content/posts/2022-01-21-scm-manager-2-30-0/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: SCM-Manager 2.30.0
+date: 2022-01-21T17:00:00+0000
+author: Eduard Heimbuch
+categories:
+- scm-manager
+- release
+- security
+keywords:
+- vulnerability
+- compare
+- autocomplete
+---
+
+Hey SCM-Manager Community,
+
+you may have read our last blog post warning you about a Path Traversal vulnerability in SCM-Manager.
+If you haven't already, please update your instance to the latest SCM-Manager version as soon as possible to fix this issue.
+
+## Alerting
+We have already been working on an alert system that will warn you directly in your instance when an instance to warn you when a vulnerability is found that affects your installed SCM-Manager core and plugin versions.
+To make sure you don't miss any important patches, each user is notified of existing security vulnerabilities.
+These alerts in the header will only appear if there are known vulnerabilities that affect your installed versions.
+
+![Alerts](assets/alerting.png)
+
+## Compare
+Another code related feature we have added is 'Compare'. 
+You can now compare diffs and changesets between two branches, tags or revisions. 
```

Comment:
"Compare diffs" sounds strange. "Create diffs" would be better, but this does not fit to the following changesets 🤔